In [4]:
from random import*
n=int(input("Cuantos lanzamientos:  "))
c=0
for i in range(n):
    d1=randint(1,6)
    d2=randint(1,6)
    if d1==d2:
        c=c+1
print("Dados iguales,en **intentos**: ",c)

Cuantos lanzamientos:  10000
Dados iguales,en **intentos**:  1690


### Lazamiento de dos dados
En este ejemplo lanzaremos dos dados, y veremos cuantos lanzamientos se necesita para que estos dados den como rresultado numeros iguales.

 - En el primer caso lanzamos 10 veces, el resultado es que en 1 intento se repitieron los resultados de los dados.
 - En el segundo caso lanzamos 100 veces, el resultado es que en 7 intento se repitieron los resultados de los dados.
 - En el tercer caso lanzamos 1000 veces, el resultado es que en 160 intento se repitieron los resultados de los dados.
 - En el cuarto caso lanzamos 10.000 veces, el resultado es que en 1690 intento se repitieron los resultados de los dados.